helm/alfresco-content-services/README.md

```diff
@@ -35,6 +35,7 @@ Hence, setting up explicit Container memory and then assigning a percentage of i
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | activemq.enabled | bool | `true` |  |
+| activemq.nodeSelector | object | `{}` |  |
```

Comment:
add a brief description to make clear what is this about?

Reply:
I added, but only to activemq.NodeSelector, cuz there is a need to change it in values.yml and i don't wanna spam 15times same sentence there :)